src/Layers/xrRenderPC_GL/r2_R_sun.cpp

```diff
@@ -300,16 +300,23 @@ struct DumbClipper
             EFC_Visible res = frustum.testAABB(&bb.vMin.x, mask);
             switch (res)
             {
-            case fcvNone: continue;
+            case fcvNone:
+            case fcv_forcedword:
+            {
```

Comment:
These parentheses are excessive and not needed in this entire diff.
Also, default case and missing cases are allowed to be missing. I believe, this warning should be disabled.